lib/6to5/transformation/transform.js

```diff
@@ -80,13 +80,12 @@ _.each({
   constants:                 require("./transformers/es6-constants"),
   letScoping:                require("./transformers/es6-let-scoping"),
 
-  _blockHoist:               require("./transformers/_block-hoist"),
```

Comment:
Put this back and I'll fix it once I've merged as I'm confident that I can fix it.